src/resultsdisassemblypage.h

```diff
@@ -0,0 +1,79 @@
+/*
+  resultsdisassemblypage.h
+
+  This file is part of Hotspot, the Qt GUI for performance analysis.
+
+  Copyright (C) 2017-2020 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+  Author: Nate Rogers <nate.rogers@kdab.com>
```

Comment:
:) also wrong

Reply:
I have added TODO for 'needs to be extracted into a standalone function, covered by unit tests and made async' to commit e2cf15317f13db10516472d8b08e04f9b0f5644f